src/components/GrowlNotification/index.js

```diff
@@ -92,10 +92,10 @@ class GrowlNotification extends Component {
                     <GrowlNotificationContainer translateY={this.state.translateY}>
                         <TouchableWithoutFeedback onPress={this.fling}>
                             <View style={styles.growlNotificationBox}>
+                                <Icon src={types[this.state.type].icon} fill={types[this.state.type].iconColor} />
```

Comment:
This change was [requested](https://github.com/Expensify/Expensify/issues/195474#issuecomment-1055840506) to keep growls consistent with the report footer (exclamation on the left).